src/FLEx-ChorusPlugin/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="icu.net" version="2.0.0" targetFramework="net40" />
+  <package id="Icu4c.Win.Min" version="54.1.31" targetFramework="net40" />
+</packages>
```

Comment:
I saw that we stopped downloading NDesk.DBus but I haven't seen a NuGet package for it yet...

Reply:
Good point. I think I confused things and therefore deleted the download. In the lfmerge branch we're still using the download - it was in Chorus where we switched to the NuGet package.

But IMO now is as good a time as any to start using the NuGet package. I'll change it.

Reply:
I repent. NDesk.DBus is an indirect dependency that's needed by SIL.Core. So the question arises to which project would we add the NDesk.DBus nuget package, and how does would it get copied to the output directory. Until we have a nuget package for SIL.Core it's simpler to get it from TC together with the SIL.Core binaries.